ocaml/xapi/xapi_vgpu_type.ml

```diff
@@ -363,7 +363,18 @@ module Vendor_nvidia = struct
   let pt_when_vgpu = true
   let whitelist_file () = !Xapi_globs.nvidia_whitelist
   let device_id_of_conf conf = conf.identifier.Identifier.pdev_id
-
+  
+  let get_host_driver_version () = try Scanf.sscanf (Unix.readlink !Xapi_globs.nvidia_host_driver_file) "libnvidia-vgpu.so.%s" (fun x -> x)
+    with _ -> info "Can not get the file version of %s, use the default value" !Xapi_globs.nvidia_host_driver_file; Xapi_globs.nvidia_default_host_driver_version
+
+  let is_host_driver_support_multi_vgpu ~host_driver_version ~supported_versions =
```

Comment:
`is_host_driver_support_multi_vgpu` is grammatically awkward. How about `host_driver_supports_multi_vgpu`?
Avoid overlong lines in new code.